app/components/admin/editions/editorial_remark_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<li class="app-component-editions-editorial-remark__list-item govuk-!-margin-bottom-4">
+<li class="app-c-editions-editorial-remark__list-item govuk-!-margin-bottom-4">
```

Comment:
This isn't a GOV.UK component, this is a view component. Should it still have an `app-c` prefix?

If I remember right we were going to style these based on views so I'd have thought it should be using `app-view` ? whereas only things in `app/views/components` would use `app-c` since they're for GOV.UK components.

Reply:
More info in here: https://github.com/alphagov/whitehall/issues/6954

Reply:
We had a long discussion about this the other day in the dev catchup. There was some discussion about maybe using something like "app-vc-*", or `app-c`, or whether they'd live in the views folder.

Since view components can live in multiple views, i think it's better for them to have their own folder rather than living in `stylesheets/admin/view_components/_component-name.scss`

Reply:
We could move it to its own namespace, which may be better for differentiating between the 2 different types of components.

Reply:
As I recall the agreement was that view components were always to be coupled to views and if it was shared it should use a regular publishing component, thus we shouldn't have the shared view component concern (and the purpose collision that has with publishing components). 

It looks like we only have one now and I imagine that should probably be just a helper since it's just a wrapper around a publishing component. So maybe we'll still safe to return to that intention?

I don't mind too much if they have a different namespace to app-view (though I think that should be fine if we stick with the view coupling intention), but do think we should avoid app-c since it'd imply it's a publishing component.

Reply:
Should we have them in all under `/views` as we're only going to be using view components for views?

Where we have shared partials, we can store them under `/views/shared`.

The less we use the word `component`, the less likely there will be confusion.

In terms of the namespace, it might be good to keep the work `component` out of it. My suggestion would be to just put everything under `app-view-*` (as all views eventually will become view components

Reply:
> @davidgisbey: Since view components can live in multiple views

> @kevindew: As I recall the agreement was that view components were always to be coupled to views and if it was shared it should use a regular publishing component

I think there's a subtle distinction here that might be helpful to acknowledge.

Our use of View Components is coupled to the View layer, but I don't think they have to be tied directly to one single 'view' file (i.e. one page). I think they could also be used in place of partials – i.e. reusable chunks of the View layer that might appear in more than one place – and still fit within our definition of acceptable use of View Components.

So another way to think about this CSS class namespacing problem is: what namespace should we be using for shared view partials? Because the same answer should probably apply to our View Components.

Reply:
Yup good shout. I think if you were to do a partial e.g. `app/views/admin/editions/_change-notes.html.rb` you'd probably use a prefix that referred to that partial e.g. `app-view-editions-change-notes` ?

Reply:
I have updated the documentation